recipes/cern-root/all/conanfile.py

```diff
@@ -0,0 +1,315 @@
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "CERN ROOT data analysis framework."
+    topics = ("data-analysis", "physics")
+    settings = ("os", "compiler", "build_type", "arch")
+    options = {
+        # Don't allow static build as it is not supported
+        # see: https://sft.its.cern.ch/jira/browse/ROOT-6446
+        # TODO: shared option should be reinstated when hooks issue is resolved
+        # (see: https://github.com/conan-io/hooks/issues/252)
+        # "shared": [True],
+        "fPIC": [True, False],
+        "python": PythonOption.ALL,
+    }
+    default_options = {
+        # "shared": True,
+        "fPIC": True,
+        # default python=off as there is currently no libpython in Conan center
+        "python": PythonOption.OFF,
+    }
+    generators = ("cmake", "cmake_find_package")
+    requires = (
+        "opengl/system",
+        "libxml2/2.9.10",
+        "glu/system",
+        "xorg/system",
+        "sqlite3/3.34.0",
+        "libjpeg/9d",
+        "libpng/1.6.37",
+        "libcurl/7.74.0",
+        "pcre/8.44",
+        "xz_utils/5.2.5",
+        "zstd/1.4.8",
+        "lz4/1.9.3",
+        "glew/2.1.0",
+        "openssl/1.1.1i",
+        "fftw/3.3.8",
+        "cfitsio/3.490",
+        "tbb/2020.3",
+        "libpng/1.6.37",
+    )
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    @property
+    def _minimum_cpp_standard(self):
+        return 11
+
+    @property
+    def _minimum_compilers_version(self):
+        return {
+            "Visual Studio": "16.1",
+            "gcc": "4.8",
+            "clang": "3.4",
+            "apple-clang": "5.1",
+        }
+
+    def configure(self):
+        if self.settings.compiler.get_safe("cppstd"):
+            tools.check_min_cppstd(self, self._minimum_cpp_standard)
+        min_version = self._minimum_compilers_version.get(str(self.settings.compiler))
+        if not min_version:
+            self.output.warn(
+                "{} recipe lacks information about the {} compiler support.".format(
+                    self.name, self.settings.compiler
+                )
+            )
+        else:
+            if tools.Version(self.settings.compiler.version) < min_version:
+                raise ConanInvalidConfiguration(
+                    "{} requires C++{} support. The current compiler {} {} does not support it.".format(
+                        self.name,
+                        self._minimum_cpp_standard,
+                        self.settings.compiler,
+                        self.settings.compiler.version,
+                    )
+                )
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("root-{}".format(self.version.replace("v", "")), self._source_subfolder)
```

Comment:
```suggestion
        os.rename("root-{}".format(self.version.replace(".", "-")), self._source_subfolder)
```

If version is change to X.Y.Z format

Reply:
Resolving conversation without implementing as ROOT versioning != sem ver. See explanation at <https://github.com/conan-io/conan-center-index/pull/3732#discussion_r562524249>.